elide-datastore/elide-datastore-hibernate3/src/main/java/com/yahoo/elide/datastores/hibernate3/HibernateTransaction.java

```diff
@@ -206,10 +206,10 @@ public void commit() {
 
         final Pagination pagination = filterScope.hasPagination()
                 ? filterScope.getRequestScope().getPagination()
-                : null;
+                : Pagination.getDefaultPagination();
 
         // if we have sorting and sorting isn't empty, then we should pull dictionary to validate the sorting rules
-        Set<Order> validatedSortingRules = null;
+        List<Order> validatedSortingRules = null;
```

Comment:
Changed it to preserve the ordering of the sorting rules. I'll change it to ordered set instead